platform-modules/mimetype-manager/src/main/scala/org/sunbird/mimetype/mgr/impl/PluginMimeTypeMgrImpl.scala

```diff
@@ -58,4 +59,8 @@ class PluginMimeTypeMgrImpl(implicit ss: StorageService) extends BaseMimeTypeMan
 			Map[String, AnyRef]("semanticVersion" -> version, "targets" -> targetList)
 		}else Map[String, AnyRef]()
 	}
+
+	override def review(objectId: String, node: Node)(implicit ec: ExecutionContext, ontologyEngineContext: OntologyEngineContext): Future[Map[String, AnyRef]] = {
+		Future(getEnrichedMetadata(node.getMetadata.getOrDefault("status", "").asInstanceOf[String]))
```

Comment:
For plugin we need to have the artifactUrl validation. Please check and enable it.

Reply:
done